src/index.js

```diff
@@ -13,5 +15,23 @@ function render() {
 		.map(result => `<tr><td>${fuzzysort.highlight(result)}</td><td>${result.obj.version}</td></tr>`)
 		.join("");
 }
+
+function updateQueryString() {
+	queryString.set('q', input.value);
+
+	const newLocation = `${window.location.pathname}?${queryString.toString()}`;
+	history.pushState(null, '', newLocation);
+}
+
+if (search) {
+	input.value = search;
```

Comment:
Maybe it would be good to have this run on the `popstate` event as well?

Reply:
I'm happy to add this to a `popstate` handler, but would the `popstate` event still be relevant if we switch to `History.replaceState()`?

Reply:
Reading the `popstate` docs more closely, it seems it would not.